requirements.txt

```diff
@@ -7,4 +7,3 @@ django-sortedm2m>=1.5.0,<1.6
 django-reversion>=2.0.6,<3.1.0
 django-x509>=0.4.0,<0.5.0
 django-taggit>=0.23.0,<0.24.0
-openwisp-utils>=0.2,<0.3
```

Comment:
we cannot remove this from here or we'll run into issues, users installing it via pypi won't get openwisp-utils installed, we have to keep it